AWS/SecretsManager/SecretRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ToucanTech\AWS\SecretsManager;
+
+use Aws\SecretsManager\Exception\SecretsManagerException;
+use Aws\SecretsManager\SecretsManagerClient;
+
+class SecretRepository {
+
+    /**
+     * Always pin the version of AWS APIs rather than using "latest" to avoid nasty shocks
+     */
+    public const SECRETSMANAGER_VERSION = '2017-10-17';
+
+    private $secretsManagerClient;
+
+    public function __construct(SecretsManagerClient $secretsManagerClient) {
+        $this->secretsManagerClient = $secretsManagerClient;
+    }
+
+    public function getSecret(string $key) {

```

Comment:
:construction: This method is simply a passthrough in its existing context and serves no real benefit within then current class, you could amend fetchSecret to public and call directly.